jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -892,21 +968,28 @@ ecma_delete_property (ecma_object_t *object_p, /**< object */
 
     ecma_property_pair_t *prop_pair_p = (ecma_property_pair_t *) cur_prop_p;
 
-    for (int i = 0; i < ECMA_PROPERTY_PAIR_ITEM_COUNT; i++)
+    for (uint32_t i = 0; i < ECMA_PROPERTY_PAIR_ITEM_COUNT; i++)
```

Comment:
Long story. The ecma_property_hashmap_delete had a different interface before. I can change it back to int.